ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_averages_and_variances.py

```diff
@@ -160,3 +160,49 @@ def test_numpy_std(
         keepdims=keep_dims,
         where=where,
     )
+
+
+# average
+@handle_frontend_test(
+    fn_tree="numpy.average",
+    dtype_and_a=statistical_dtype_values(function="average"),
+    dtype_and_x=statistical_dtype_values(function="average"),
+    keep_dims=st.booleans(),
+)
```

Comment:
Just noticed the `returned` argument is not tested. Could you please add it here?